docs/_tutorials/ja_getting_started.md

```diff
@@ -152,7 +150,10 @@ ngrok http 3000
 **Request URL**  ボックスの  **Enable Events**  スイッチの下のフィールドにこの URL を貼り付けます。Bolt アプリが引き続き実行されている場合は、URL が検証されチェックマークが表示されます。
 
 Request URL が検証されたら、**Subscribe to Bot Events** までスクロールします。メッセージに関するイベントが４つあります−
```

Comment:
@mwbrooks Thanks! We do not have any habits to use `-` over `:` here in Japanese too. We should change the part this way.
```suggestion
Request URL が検証されたら、**Subscribe to Bot Events** までスクロールします。メッセージに関するイベントが４つあります:
```

Reply:
Awesome, thanks! Added your suggestion!